### DISCUSSION TITLE
Welcome to OMOP2OBO Discussions!

### DISCUSSION THREAD

**Discussion by callahantiff** [callahantiff/OMOP2OBO#54] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ramonawalls** [comment]
Hi @callahantiff . Thanks for this great resource! I am planning a knowledge graph for rare disease non-genomic data as part of the [RDCA-DAP](https://c-path.org/programs/rdca-dap/) that I hope will be complementary to projects like Monarch Initiative. We are looking at using biolink as an upper model, which has mappings to some OBO ontologies, but we also need mappings to OMOP CDM. This repository looks like it will be incredibly valuable to our project, and I hope we can contribute back.

**Reply by callahantiff** [reply]
Hi @ramonawalls - Thank you so much for considering `OMOP2OBO` for your project, I'm thrilled! 😄  I'd love to learn more about your use case and how we can support the work that you are doing.  I'd also be more than happy to set-up a meeting where I can provide more details about the mapping algorithm and validation we have performed (in the process of finishing up now) as well as hear more about your plans.

**Reply by ramonawalls** [reply]
Thank you @callahantiff . We are working with data from registries, clinical trials, and EHR. Our first step is to convert data to OMOP CDM. I think our use case for your tools will be to map our OMOP data to OBO ontologies. We may also be able to use PheKnowLater. I would love to have a call with you to discuss your tools and how we can make use of them, if you have a half hour or hour to spare. You can reach me at rwalls at c-path.org.

**Reply by callahantiff** [reply]
Hi @ramonawalls! That sounds great, I will follow up by email.

**Comment by hms1** [comment]
Hi @callahantiff,

Thank you for creating this tool and for the effort you have put in to making things clear and easy to get started with 😄 .

I'd be interested learn more about what you have done/ are planning and more importantly if there is anything that you need help with. I have a lot of experience working with OMOP data and more recently have been looking into the mapping of phenotypic data for omics work, so am particularly interested in your work mapping to _hp_.

Thanks again for the resource!

Henry

**Reply by callahantiff** [reply]
Hi @hms1 -

It's a pleasure to meet you. Thank you so much for your interest in OMOP2OBO.  🙌 🥳

I'd be very happy to discuss this work further (especially the work to align with HP) as well as potential opportunities to work together! Your work sounds very interesting and mapping expertise would be incredibly valuable. I'd also love to learn more about your work. I'm working on a new release that should make substantial improvements to the underlying architecture and as a result, improve the output mappings. I'd love to tell you more about this and show you some preliminary results for HP.

Would you be willing to share your email address with me (or email me directly at callahantiff@gmail.com)? I'd love to find a time for us to meet and discuss all of this further. Does that sound OK to you?

Thanks again for reaching out. I am looking forward to speaking with you!


-Tiffany
